x/migration/recovery/recovery_allowlist.go

```diff
@@ -0,0 +1,2071 @@
+package recovery
+
+var recoveryAllowlist = []string{
+	// Lost AppStakes
```

Comment:
Rather than having comments for different sections, can you:

1. Split this into separate, semantically named, lists
2. Concat them via an `init` (or something similar)?